test/e2e/specs/fse/fse__limited-global-styles.ts

```diff
@@ -12,8 +13,12 @@ declare const browser: Browser;
  * style variations (eg. Twenty Twenty-Three).
  *
  * @see https://github.com/Automattic/wp-calypso/issues/78107
+ *
+ * We skip Atomic sites because they are not affected by Limited Global Styles.
+ *
+ * @see https://github.com/Automattic/wp-calypso/pull/71333#issuecomment-1592490057
  */
-describe( 'Site Editor: Limited Global Styles', function () {
+skipDescribeIf( envVariables.TEST_ON_ATOMIC )( 'Site Editor: Limited Global Styles', function () {
```

Comment:
If we add this test we can implicitly test that the code is working as intended, since otherwise the test would be failing. With this change, if the build completes we can guarantee that your changes are working.

Besides this, the code looks good and I don't think there should be any problems (since the build for this PR should have failed if there were).

```suggestion
skipDescribeIf( envVariables.TEST_ON_ATOMIC )( 'Site Editor: Limited Global Styles', function () {
    test( 'Site should not be atomic', () => {
		expect( envVariables.TEST_ON_ATOMIC ).toBeFalsy();
	} );

```